index.bs

```diff
@@ -3657,7 +3664,86 @@ This is because the first 37 bytes of the signed data in a FIDO U2F authenticati
 <code>[=authDataExtensions|extensions=]</code> are never present. FIDO U2F authentication signatures can therefore be verified by
 the same procedure as other [=assertion signatures=] generated by the [=authenticatorGetAssertion=] operation.
 
+### Credential Backup State ### {#sctn-credential-backup}
 
+Credential backup eligibility and current backup state is conveyed by [=Authenticator data|authenticator data=] bits `3` and `4` as 
+defined in <a href="#table-authData">Table <span class="table-ref-following"/></a>.
+
+The values of bits 3 and 4 may change over time based on the state of the [=authenticator=]. <a href="#table-backupStates">Table <span class="table-ref-following"/></a> below defines
+valid combinations and their meaning.
+
+<figure id="table-backupStates" class="table">
+    <table class="complex data longlastcol">
+            <tr>
+            <th>Bit 3 value</th>
+            <th>Bit 4 value</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td>`0`</td>
+            <td>`0`</td>
+            <td>
+                The credential is a single-device credential and cannot become a multi-device credential
+            </td>
+        </tr>
+        <tr>
+            <td>`0`</td>
+            <td>`1`</td>
+            <td>
+                This combination is not allowed
```

Comment:
The fact that the bits can "change over time" is a red flag, since authenticator data is only sent at registration, not auth, which means when the bits change, the RP won't know they changed.

Reply:
> The fact that the bits can "change over time" is a red flag, since authenticator data is only sent at registration, not auth, which means when the bits change, the RP won't know they changed. 

authData is sent during a getAssertion and the RP can evaluate any change to backup state.

Reply:
Yes, my mistake, I thought this was in the attestation. 

In this case, I think the issue is that there is no advie from the standard that an RP should persist the authdata flags to detect changes (this has previously caused CVE's related UV anyway ...)

Reply:
> Yes, my mistake, I thought this was in the attestation.
> 
> In this case, I think the issue is that there is no advie from the standard that an RP should persist the authdata flags to detect changes (this has previously caused CVE's related UV anyway ...)

Covered in section 7 with the other verification procedures.